dockers/docker-base/etc/supervisor/supervisord.conf

```diff
@@ -17,6 +20,8 @@ supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
 
 [supervisorctl]
 serverurl=unix:///var/run/supervisor.sock ; use a unix:// URL  for a unix socket
+username=nobody
+password=nobody
```

Comment:
from your commit message, you are trying to address http server accessing issue, so I am not sure if we need to add username/password for the supervisorctl command line as well. Since once you can get into the docker, you already have sudo privilege, adding extra user/pass protection does not seem necessary.